lib/business_time/core_ext/date.rb

```diff
@@ -2,15 +2,18 @@
 class Date
   include BusinessTime::TimeExtensions
   include BusinessTime::Currency
-  
-  def business_days_until(to_date, *currency)
+
+  def business_days_until(to_date, inclusive = false, *currency)
     currency = args(*currency)
-    business_dates_until(to_date, *currency).size
+    business_dates_until(to_date, inclusive, *currency).size
   end
 
-  def business_dates_until(to_date, *currency)
+  def business_dates_until(to_date, inclusive = false, *currency)
```

Comment:
I don’t like this solution, but let’s have it as is unless we invent a use-case.